css/index.css

```diff
@@ -1,3 +1,43 @@
 body {
   background: #fff;
+  text-align: center;
+  font-family: cursive;
+}
+
+#books-list {
+  border: 1px solid black;
+  margin-bottom: 32px;
+}
+
+#books-list > div {
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5% 3%;
+}
+
+#books-list > div:nth-child(odd) {
+  background-color: rgb(221, 218, 215);
+}
+
+.separator {
+  margin: 16px;
+  font-size: 14px;
+}
+
+.add-books-form {
+  display: flex;
+  flex-direction: column;
+  font-size: larger;
+  margin: 0 35%;
+}
+
+#submitBtn {
+  align-self: flex-end;
+  width: 20%;
+}
```

Comment:
- [ ] I would suggest keeping more gap between the inputs and the button as well to match the design and also to make UI more awesome.

### Your design 

![Screenshot from 2021-10-19 19-47-01](https://user-images.githubusercontent.com/22434660/137929228-892e9026-6b8c-4d2e-8cb5-bb1fd7599c36.png)


### GIven design

![Screenshot from 2021-10-19 19-49-01](https://user-images.githubusercontent.com/22434660/137929307-43d089e6-344f-404c-9f3b-ebc2f7a327c5.png)